README.md

```diff
@@ -249,6 +283,182 @@ The following table lists available configuration parameters.
+for this operator is that for each workload there are security reports stored in the cluster as custom resources.
+
+Currently, the operator implements two reconciliation loops and only supports [vulnerabilityreports][vulnerabilityreports-crd]
+security resources as depicted below. However, we plan to support all [custom security resources][starboard-crds].
+
+| Controller | Description |
+| ---------- | ----------- |
+| [PodController](pkg/operator/controller/pod/pod_controller.go) | Watches for pod events in target namespaces to lookup the immediate owner of a pod. Then it checks whether there's the VulnerabilityReport owned by this owner. If not, it schedules a scan job in the operator's namespace. |
+| [JobController](pkg/operator/controller/job/job_controller.go) | Watches for job events in the operator's namespace. If a given job is completed it parses the logs of the controlee pod and converts the logs output to an instance of the VulnerabilityReport resource. |
+
+![](docs/images/operator/starboard-operator.png)
+
+### Deployment
+
+#### With Static YAML Manifests
+
+You can install the operator with provided static YAML manifests with fixed values. However, this approach has its
+shortcomings. For example, if you want to change the container image or modify default configuration parameters, you
+have to create new manifests or edit existing ones.
+
+To deploy the operator in the `starboard-operator` namespace and configure it to watch the `default`
+namespace:
+
+1. Send the definition of the [vulnerabilityreports][vulnerabilityreports-crd] custom resource to the Kubernetes API:
+
+   ```
+   $ kubectl apply -f deploy/crd/vulnerabilityreports.crd.yaml
+   ```
+2. Send the following Kubernetes objects definitions to the Kubernetes API:
+
+   ```
+   $ kubectl apply -f deploy/static/01-starboard-operator.ns.yaml \
+       -f deploy/static/02-starboard-operator.sa.yaml \
+       -f deploy/static/03-starboard-operator.clusterrole.yaml \
+       -f deploy/static/04-starboard-operator.clusterrolebinding.yaml
+   ```
+3. Create the `starboard-operator` deployment in the `starboard-operator` namespace to run the operator's container:
+
+   ```
+   $ kubectl apply -f deploy/static/05-starboard-operator.deployment.yaml
+   ```
+
+#### With Helm
+
+[Helm][helm], which is de facto standard package manager for Kubernetes, allows installing applications from
+parameterized YAML manifests called Helm [charts][helm-charts].
+
+To address shortcomings of static YAML manifests we provide the Helm chart to deploy the Starboard operator. The
+[starboard-operator](./deploy/helm) Helm chart supports all [install modes](#install-modes). For example, to install
+the operator in the `starboard-operator` namespace and configure it to watch `foo` and `bar` namespaces, run:
+
+```
+$ kubectl create ns starboard-operator
+$ helm install starboard-operator -n starboard-operator \
+    ./deploy/helm \
```

Comment:
There is also the --create-namespace flag btw

Reply:
Great. I didn't know that. Thanks for letting us know. I used the flag and merged the two commands into one!